src/pynwb/misc.py

```diff
@@ -232,7 +232,7 @@ def __init__(self, **kwargs):
              'doc': 'the index of the unit in unit_ids to retrieve spike times for'})
     def get_unit_spike_times(self, **kwargs):
         index = getargs('index', kwargs)
-        return self.__iv.get_vector(index)
+        return np.array(self.__iv.get_vector(index))
```

Comment:
This could create some unwanted overhead down the road if spike times gets large, but I don't want to let perfection stand in the way of progress.